electron/config.ts

```diff
@@ -239,15 +240,14 @@ abstract class GlobalConfig {
       })
     }
 
+    const steamLibraries = getSteamLibraries()
     const protonPaths = [`${heroicToolsPath}/proton/`]
 
     // Known places where Steam might be found.
     // Just add a new string here in case another path is found on another distro.
-    const steamPaths = [
-      join(userHome, '.steam'),
-      join(userHome, '.var/app/com.valvesoftware.Steam/.local/share/Steam'),
-      '/usr/share/steam'
-    ].filter((path) => existsSync(path))
+    const steamPaths = [...steamLibraries, '/usr/share/steam'].filter((path) =>
```

Comment:
Why not just call `getSteamLibraries()` directly here?

Reply:
Yes, still doesn't make any difference in practice :P

Reply:
Well, one less variable to worry about

Reply:
Did a refactor to put most of the work on the helper function now, check if it looks better.